### PR TITLE
Typo "Teams Room app"→"Teams Rooms app"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/microsoftteams/rooms/enroll-a-device
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/rooms/enroll-a-device.md
#PingMSFTDocs

### CODE REVIEW

File: Teams/rooms/enroll-a-device.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ To log in to the device as Administrator (local administrator):
 > [!NOTE]
 > If the computer is *domain joined*, choose **Other User**, then use **.\admin**, or the user name of the local administrator configured in the device as the user name.
 
-To return to the Microsoft Teams Room app after performing the necessary administrative tasks:
+To return to the Microsoft Teams Rooms app after performing the necessary administrative tasks:
 
 1. From the Windows ***Start menu***, sign out from the Admin account.
 1. Return to Microsoft Teams Room by selecting the user account icon on the far left side of the screen and then selecting **Skype**.
```
Comment (JohanFreelancer9) [inline Teams/rooms/enroll-a-device.md:88]:
```suggestion
1. Return to Microsoft Teams Rooms by selecting the user account icon on the far left side of the screen and then selecting **Skype**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested change to also correct the typo in Line 88. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@donnah007 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by donnah007** [comment]
Merged, thanks for the copyedit.

**Pull request merged**